demo/index.js

```diff
@@ -220,11 +221,36 @@ $(function() {
 
     }
 
+    function addTrack(session){
+            var pc = session.sessionDescriptionHandler.peerConnection;
+
+            // Gets remote tracks
+            var remoteVideo = document.getElementById('remoteVideo');
+            var remoteStream = new MediaStream();
+            pc.getReceivers().forEach(function(receiver) {
+                remoteStream.addTrack(receiver.track);
+            });
+            remoteVideo.srcObject = remoteStream;
+            remoteVideo.play();
+
+            // Gets local tracks
+            var localVideo = document.getElementById('localVideo');
+            var localStream = new MediaStream();
+            pc.getSenders().forEach(function(sender) {
+                localStream.addTrack(sender.track);
+            });
+            localVideo.srcObject = localStream;
+            localVideo.play();
+    }
+
```

Comment:
Remove this from demo completely, put this inside the SDK as a part of session patching procedure. SDK constructor should receive two references to `localVideo` and `remoteVideo`.